src/main/java/objectsBasicsMandatoryPracticalTasks/Book.java

```diff
@@ -0,0 +1,17 @@
+package objectsBasicsMandatoryPracticalTasks;
+
+class Book {
+    private String title;
+    private double price;
+    Author author;
```

Comment:
Author should be private as well so that other classes won't modify author directly